### ISSUE TITLE
non-ascii parametrized arguments throw `UnicodeDecodeErrors` as of 2.8.1

### ISSUE THREAD

**Issue by hummus** [pytest-dev/pytest#1107]
non-ascii parametrized arguments throw `UnicodeDecodeErrors` as of 2.8.1.

###### test_unicode_parametrize.py:
```
# -*- coding: utf-8 -*-
import pytest

@pytest.mark.parametrize(u'foo', (u'ç',))
def test_foo(foo):
    assert True
```

###### to reproduce:
```
$ pip install pytest==2.8.0
$ py.test -x test_unicode_parametrize.py
....
===== 1 passed
$ pip install pytest==2.8.1
$ py.test -x test_unicode_parametrize.py
E   UnicodeEncodeError: 'ascii' codec can't encode character u'\xe7' in position 0: ordinal not in range(128)
```

**Comment by The-Compiler** [comment]
This should be fixed in #1092 and already has a handful of duplicates :wink:

Can you verify it works with your testsuite when running the current master?

```
pip install git+https://github.com/pytest-dev/pytest
```

there should be a 2.8.2 release with the fix early next week.

**Comment by hummus** [comment]
sorry should have checked that first! master is fine.
